ansible/roles/operator-pipeline/templates/openshift/tasks/add-fbc-fragments-to-index.yml

```diff
@@ -0,0 +1,105 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: add-fbc-fragments-to-index
+spec:
+  params:
```

Comment:
Can you add a `.spec.description` field to the two new tasks to explain what each task does? So far we have not been very consistent in doing this but, since we have no real documentation about each task, this helps a bit. See the buildah task for an example.

Reply:
Description added